src/concepts/glean/accessing_glean_data.md

```diff
@@ -0,0 +1,71 @@
+# Accessing Glean data
+
+This document describes how to access Glean data from an SQL query, such as in [Redash](https://sql.telemetry.mozilla.org).
+
+## Selecting the correct table
+
+Each ping type is recorded in its own table, and these tables are named using `{application_id}.{ping_type}`.
+For example, for Fenix, the application id is `org_mozilla_fenix`, so its `metrics` pings are available in the table `org_mozilla_fenix.metrics`.
+
+## Selecting columns
+
+Columns of data are named using dot notation to organize them into groups.
+For example, columns containing Glean's built-in client information all begin with the `client_info.` prefix.
+
+The top-level groups are:
+
+* `client_info`: [Client information provided by Glean](https://mozilla.github.io/glean/book/user/pings/index.html#the-client_info-section).
+* `ping_info`: [Ping information provided by Glean](https://mozilla.github.io/glean/book/user/pings/index.html#the-ping_info-section).
```

Comment:
I am actively working on https://github.com/mozilla/gcp-ingestion/issues/633 right now which will make new `ping_info.parsed_start_time` and `ping_info.parsed_end_time` fields available in the user-facing views for glean pings. We can have a separate docs PR once that is change is merged and deployed, but this definitely seems like a good place to advertise the existence of those parsed fields.

Reply:
Agreed!  Let's add those here when that lands.

Also, this makes me realize there are other fields that are beyond the ping content that should probably be documented here, such as `submission_timestamp` and `document_id`.